store/blocks/container-mobile.json

```diff
@@ -0,0 +1,29 @@
+{
+  "flex-layout.row#container-mobile": {
+    "children": ["flex-layout.col#checkout-mobile"],
+    "props": {
+      "blockClass": "checkoutContainerMobile",
+      "paddingTop": 6
+    }
+  },
+
```

Comment:
Is this empty line intentional?

Reply:
yes, it was to visually separate the blocks, but it doesn't help much. I just find it too difficult to understand the structure of the page by looking at JSON files and tried to separate them better, but its still really awful at visualizing it :-(

Reply:
but I can remove it if you want!

Reply:
No, that's fine. I'm not well versed on "blocks", I have the same difficulties to understand these giant JSONs but want to make sure if there is any type of standard related to empty lines.